### PR TITLE
Fix source dragging on app-start

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

**Comment by andycreeth** [comment]
Looking at the history, it looks like this flag was added along with source grouping.  I'm going through the code and I honestly don't understand what the purpose of this flag is.  Can you maybe explain why we need this?

**Comment by holiber** [comment]
Didn't remember. Will take a look

File: app/components/StudioEditor.vue.ts [hunk 1]
```diff
@@ -87,13 +87,14 @@ export default class StudioEditor extends Vue {
       }
     }
 
+    // prevent dragging if the clicking is past the source
+    if (!this.getOverSource(event)) this.canDrag = false;
```
Comment (holiber) [inline app/components/StudioEditor.vue.ts]: I found this bug, so added a comment here to not to forget

**Comment by andycreeth** [comment]
Any idea why the tests are failing?  Can we fix?

**Pull request merged**